docs/guides/advanced/execute_remotely.md

```diff
@@ -52,8 +52,7 @@ Logger.current_logger().report_scalar(
 )
 ```    
 
-These scalars can be visualized in plots, which appear in the ClearML web UI, in the experiment's
-page **>** **SCALARS**. 
+These scalars can be visualized in plots, which appear in the ClearML web UI, in the experiment's **SCALARS**. 
```

Comment:
@pollfly scalars **tab**?

Reply:
added "tab"s, also in new commit switched page >>> tab when describing experiment sections